src/libs/sfdk/virtualmachine_p.h

```diff
@@ -192,6 +192,7 @@ class VirtualMachinePrivate
     QSsh::SshConnectionParameters sshParameters;
     bool headless = false;
     bool autoConnectEnabled = true;
+    VirtualMachine::Features features;
```

Comment:
I would prefer moving this up before `name`. I would consider applying that order in the ctor too.

Reply:
moved